apps/video/js/video.js

```diff
@@ -1320,7 +1331,11 @@ function handleSliderTouchMove(event) {
   dom.playHead.classList.add('active');
   movePlayHead(percent);
   dom.elapsedTime.style.width = percent;
-  dom.player.fastSeek(dom.player.duration * pos);
+  if (dom.player.seeking) {
+    pendingSeekPosition = dom.player.duration * pos;
+  } else {
+    dom.player.fastSeek(dom.player.duration * pos);
+  }
```

Comment:
I'd suggest you write a fastseek() utility function that you can use here.  You should also use it below in handleSliderKeypress()